apps/state/config/config.exs

```diff
@@ -105,4 +105,15 @@ config :state, :shape,
     "-S" => -1
   }
 
+# Overrides for the stop ordering on routes where the trips themselves aren't enough
+config :state, :stops_on_route,
+  stop_order_overrides: %{
+    {"CR-Kingston", 0} => [
+      ["Hanson", "Halifax", "Kingston", "Plymouth"]
+    ],
+    {"CR-Franklin", 0} => [
+      ["Norwood Central", "Windsor Gardens", "Plimptonville", "Walpole"]
+    ]
```

Comment:
These don't need to be lists. We probably won't need more than one override per (route,direction), and even if we do, I think one longer override would probably work as well as two smaller ones.